storey/flow.py

```diff
@@ -1511,16 +1513,30 @@ async def run_async(self, body: Any, path: str) -> Any:
         return body
 
     def _run(self, body: Any, path: str) -> Any:
+        timestamp = datetime.datetime.now(tz=datetime.timezone.utc)
         start = time.monotonic()
-        body = self.run(body, path)
+        try:
+            body = self.run(body, path)
+        except Exception as e:
+            if not self._raise_exception:
+                body = {"error": str(e)}
```

Comment:
I think `str(e)` gives just the message, without the type. So something like `NotImplementedError` without a message would appear as `{"error": ""}`. That is to say, we should probably make it something like
```suggestion
                body = {"error": f"{type(e)}: {e}"}
```